### DISCUSSION TITLE
Is it possible to add color emojis into nerdfonts?

### DISCUSSION THREAD

**Discussion by v3ss0n** [ryanoasis/nerd-fonts#1076] [General]
The fonts generated are missing colored emojis , i am checking if it is possible to patch color emojis from google?

https://fonts.google.com/noto/specimen/Noto+Color+Emoji

**Comment by Finii** [comment]
I'm not sure if the Emoji set is a good addition, usually it fares ok enough as separate font. Fontfallback will solve this on all platforms nowadays, I believe ;-)

Anhow, color glyphs are a bit ... harder, see #870.

I'm not sure it is possible at all (with `fontforge`).
I would have to buy a book or two to learn how that works ;-) [1]
It would probably need a considerable amount of new code.

Why can you not work with the emoji in a separate font? I use that exact font (this is about the only Google font I have installed _at all_) and it just works.

[1] see
![image](https://user-images.githubusercontent.com/16012374/214529703-855a484c-a5a2-432c-9b33-26f24b7ca7c6.png)

_Edit: Add book reference_ :-D

**Reply by v3ss0n** [reply]
In applications without multiple font setting , how we can do that? In CSS though i can do this and it works .

```
*
{
  font-family: 'IBM Plex Sans','Noto Color Emoji',serif,sans-serif;
}

input, textarea, select
{
  font-family: 'BlexMono Nerd Font','Noto Color Emoji','monospace';
}

a, abbr, blockquote, center, canvas, iframe, li, ol, ul, p, quote, label, caption, img, input[type="submit"], input[type="button"], input[type="reset"], input[type="file"], table, th, tr, td
{
  font-family:  'IBM Plex Sans','Noto Color Emoji',serif,sans-serif;:
  ```

**Reply by Finii** [reply]
You can try https://github.com/ryanoasis/nerd-fonts/wiki/Font-Substitution-&-Fallback

That will not work with all applications, though.

And color fonts will also not work in all applications iirc :thinking:

**Reply by Cyberbeni** [reply]
I think emojis would be useful for icons that should have multiple colors, like nf-dev-python or nf-dev-xcode.
